test/views/declaration/spec/ViewSpec.scala

```diff
@@ -42,6 +49,9 @@ trait ViewSpec extends PlaySpec with GuiceOneAppPerSuite with ViewValidator {
   implicit lazy val fakeRequest: Request[AnyContentAsEmpty.type] = FakeRequest("", "").withCSRFToken
   implicit lazy val messages: Messages = messagesApi.preferred(FakeRequest())
   implicit lazy val flash: Flash = new Flash()
+  implicit protected def htmlBodyOf(html: Html): Document = Jsoup.parse(html.toString())
+  implicit protected def htmlBodyOf(page: String): Document = Jsoup.parse(page)
+  implicit protected def htmlBodyOf(result: Future[Result]): Document = htmlBodyOf(contentAsString(result))
```

Comment:
Allows us to create a view as a `HTML/String/Future[Result]` type and use JSOUP `Document` matchers without extra boilerplate